app/modules/encounters/resources.py

```diff
@@ -174,19 +95,90 @@ def get(self, encounter):
     )
     @api.login_required(oauth_scopes=['encounters:write'])
     @api.parameters(parameters.PatchEncounterDetailsParameters())
-    @api.response(schemas.DetailedEncounterSchema())
     @api.response(code=HTTPStatus.CONFLICT)
     def patch(self, args, encounter):
         """
         Patch Encounter details by ID.
         """
+
+        edm_count = 0
+        for arg in args:
+            if (
+                'path' in arg
+                and arg['path']
+                in parameters.PatchEncounterDetailsParameters.PATH_CHOICES_EDM
+            ):
+                edm_count += 1
+        if edm_count > 0 and edm_count != len(args):
+            log.error(f'Mixed edm/houston patch called with args {args}')
+            abort(
+                success=False,
+                passed_message='Cannot mix EDM patch paths and houston patch paths',
+                message='Error',
+                code=400,
+            )
+
+        rdata = {}
+        if edm_count > 0:
+            log.debug(f'wanting to do edm patch on args={args}')
+            headers = {
+                'x-allow-delete-cascade-individual': request.headers.get(
+                    'x-allow-delete-cascade-individual', 'false'
+                ),
+                'x-allow-delete-cascade-sighting': request.headers.get(
+                    'x-allow-delete-cascade-sighting', 'false'
+                ),
+            }
+            response = current_app.edm.request_passthrough(
+                'encounter.data',
+                'patch',
+                {'data': args, 'headers': headers},
+                encounter.guid,
+            )
+            rdata = response.json()
```

Comment:
This block is also duplicated.
How about creating a new EDMManager implements request_passthrough, sets the headers as above, calls the base class request_passthrough to do the processing, does the checking below and raises an exception with the required info. Something like AssetGroupMetadataError but called HoustonGenericException and implemented in app/utils.py (new file).
Calling code here would catch exception and abort with appropriate info.
The benefit of this is that the HoustonGenericException becomes an input into the AuditLog when we implement that.